clusterloader2/config/codec.go

```diff
@@ -18,13 +18,18 @@ package config
 
 import (
 	"fmt"
+	"io/ioutil"
 	"path/filepath"
 	"strings"
 
 	"github.com/spf13/viper"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/scheme"
 	"k8s.io/perf-tests/clusterloader2/api"
 )
 
+var decode = scheme.Codecs.UniversalDeserializer().Decode
```

Comment:
It seems like you only need it at one place - maybe you can directly call original function there?

Reply:
ok